II.pass-tests/ES-6-practice/es6.js

```diff
@@ -1,11 +1,18 @@
 // Turn this function into an arrow function
-function arrow() {
+// function arrow() {
+//   return "es6 is awesome!";
+// }
+
+// };
+const arrow = () => {
   return "es6 is awesome!";
 }
 
+
 //Use the es6 syntax for default parameter
-function defaultParameter(name) {
-  var name = name || "sam";
+function defaultParameter(name ) {
+  var name = typeof name !== "undefined" ? name : "sam";
```

Comment:
This isn't quite correct. This is how we did default parameters before the ES6 syntax was introduced. Now we can do this:

```js
function greeting(hello = "world") {

}
```